VictoriaAlbert/VictoriaAlbert/DetailViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  DetailViewController.swift
+//  VictoriaAlbert
+//
+//  Created by Marcel Chaucer on 11/10/16.
+//  Copyright © 2016 Marcel Chaucer. All rights reserved.
+//
+
+import UIKit
+
+class DetailViewController: UIViewController {
+     var  detailObject: MuseumObject?
+
+    @IBOutlet weak var pictureOfObject: UIImageView!
+    
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        if let detail = detailObject {
+            self.title = detail.object
+            
+            var firstSix = detail.imageID
+            let range = firstSix.index(firstSix.endIndex, offsetBy: -4)..<firstSix.endIndex
+            let smallPic = firstSix.removeSubrange(range)
+            
+            
+
+            
+            
+    let fullImage = "http://media.vam.ac.uk/media/thira/collection_images/\(firstSix)/\(detail.imageID).jpg"
+            
+            let url = URL(string: fullImage)
+            let data = try? Data(contentsOf: url!)
```

Comment:
```Data(contentsOf: url!)``` is not asynchronous. You should be using getData for all of your networking, json and images.